app/src/main/AndroidManifest.xml

```diff
@@ -135,6 +135,7 @@
         <activity
             android:name=".profile.ProfileActivity"
             android:configChanges="orientation|screenSize|keyboard"
+            android:parentActivityName=".category.CategoryDetailsActivity"
```

Comment:
Why is the parent, CategoryDetailsActivity, should it not be MainActivity?

Reply:
Please, watch the video in the drive link. The user should return back to the menu Category, isn't it?
If not then I will make the above-mentioned changes.

Reply:
Should I make the above-suggested changes?

Reply:
Hi, I think the back press should take the user, back to the screen from where this activity was opened, lets just use super.back pressed

Reply:
@Roshan13046 Do you really need to insert this line? Does the PR not work if you do not change this file at all?

Reply:
> @Roshan13046 Do you really need to insert this line? Does the PR not work if you do not change this file at all?

which code line?

Reply:
This code line: `            android:parentActivityName=".category.CategoryDetailsActivity"`